docs/specs/supporting-tools.md

```diff
@@ -0,0 +1,86 @@
+# Supporting tools and services
+
+This page outlines tools and services that currently support the development container specification, including the `devcontainer.json` format. A `devcontainer.json` file in your project tells tools and services that support the dev container spec how to access (or create) a dev container with a well-defined tool and runtime stack. 
+
+While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
+
+## Editors
+
+### Visual Studio Code
+
+Visual studio code specific properties go under `vscode` inside `customizations`.
+
+
+```
+"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {},
+			"extensions": [],
+			]
```

Comment:
Indentation looks good! Should we remove this trailing `]`?

Reply:
Fixed.